subprojects/test-kit/src/integTest/groovy/org/gradle/testkit/runner/GradleRunnerMechanicalFailureIntegrationTest.groovy

```diff
@@ -169,7 +171,8 @@ class GradleRunnerMechanicalFailureIntegrationTest extends BaseGradleRunnerInteg
         and:
         def output = OutputScrapingExecutionResult.from(t.message, "")
         def taskHeader = gradleVersion >= GradleVersion.version("4.0") ? "\n> Task :helloWorld" : ":helloWorld"
-        output.normalizedOutput == """An error occurred executing build with args '${runner.arguments.join(' ')}' in directory '$testDirectory.canonicalPath'. Output before error:
+        def actualArguments = OperatingSystem.current().windows ? ["-D${StartParameterBuildOptions.WatchFileSystemOption.GRADLE_PROPERTY}=false"] + runner.arguments : runner.arguments
```

Comment:
Let's break this into multiple lines. I think a comment with some explanation here would also be nice, it could even point to the issue or this PR, or even to `GradleRunner` that does have the comments already.

Reply:
Broken and added a comment.